hack/lib/util/golang.sh

```diff
@@ -8,10 +8,10 @@ function os::golang::verify_go_version() {
 
 	local go_version
 	go_version=($(go version))
-	if [[ "${go_version[2]}" != go1.7* ]]; then
+	if [[ "${go_version[2]}" != go1.8* ]]; then
 		os::log::info "Detected go version: ${go_version[*]}."
 		if [[ -z "${PERMISSIVE_GO:-}" ]]; then
-			os::log::error "Please install Go version 1.7 or use PERMISSIVE_GO=y to bypass this check."
+			os::log::error "Please install Go version 1.8 or use PERMISSIVE_GO=y to bypass this check."
```

Comment:
Changed to `os::log:fatal`, rebased -> cc3b0b09ca40527e443f503aa4cffa6752f73d83.